client/my-sites/post-share/style.scss

```diff
@@ -66,66 +65,28 @@ $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 }
 
 .post-share__form {
-	flex: 3;
```

Comment:
I agree with the need for a new component. But the issue around the form above is different issue.